comp/core/tagger/noopimpl/tagger.go

```diff
@@ -73,12 +73,10 @@ func (n *noopTagger) GetEntity(string) (*types.Entity, error) {
 	return nil, nil
 }
 
-func (n *noopTagger) Subscribe(types.TagCardinality) chan []types.EntityEvent {
-	return make(chan []types.EntityEvent)
+func (n *noopTagger) Subscribe(string, *types.Filter) types.Subscription {
+	return nil
```

Comment:
should you return a "noobSubscription" here or do you expect that the user is checking that the subscription is not nil?

IMO, since `Subscription` is an interface, `Subscribe()` should not return nil